### ISSUE TITLE
[1.8] Cloud height and rendering terrain

### ISSUE THREAD

**Issue by idc-github-account** [MinecraftForge/MinecraftForge#2222]
In the class EntityRenderer at line 1315 and 1452 the cloud renderer gets called but only one of those depending on where on the Y-axis the player is, if above 128 then the later gets called. This is becomes problematic if you have changed the cloud height to lower than 128. If you do so and look down on the terrain, then the clouds appear above transparent blocks when they are actually below the said transparent block.
http://i.imgur.com/Hb7P5Bh.png

**Comment by williewillus** [comment]
This is a vanilla bug, iirc.
Fixes attepmted by mojang each time have resulted in performance hits so they've just left it as is
See: https://bugs.mojang.com/browse/MC-997

**Comment by cpw** [comment]
Yeah, the lesson is 'don't mess with clouds, cos they'll mess you up!'

**Comment by killjoy1221** [comment]
I think I've found a fix, but I'm unsure of the implications of it.  Currently, cloud rendering is done twice, early if we're below the clouds, later if we're above. Above works, so I removed the above render and the check for height in the below render.

Again, I'm not sure why it was done like this, so we might want to find out why. Maybe it caused lag?

https://gist.github.com/killjoy1221/47aefe98ab1d5b71cc1c
